Sources/GRPC/AsyncAwaitSupport/AsyncWriter.swift

```diff
@@ -296,11 +296,11 @@ extension AsyncWriter where End == Void {
   }
 }
 
-public struct GRPCAsyncWriterError: Error, Hashable {
+public struct GRPCAsyncWriterError: Error, Sendable, Hashable {
```

Comment:
Sendability is already required from `Error` conformance.

Reply:
https://developer.apple.com/documentation/swift/error